schema/spacewalk/upgrade/susemanager-schema-5.0.10-to-susemanager-schema-5.0.11/010-foreign-groups.sql

```diff
@@ -0,0 +1,7 @@
+insert into rhnServerGroupTypeFeature (server_group_type_id, feature_id,
+                                       created, modified)
+select lookup_sg_type('foreign_entitled'), lookup_feature_type('ftr_system_grouping'),
+       current_timestamp, current_timestamp from dual
```

Comment:
Nitpick `from dual` is no longer needed since we no longer support oracle.

Reply:
We have it everywhere.